amplipi/ctrl.py

```diff
@@ -533,11 +533,21 @@ def _check_latest_release(self) -> str:
       pass
     return release
 
+  def get_usb_drives(self):
+    """Reads the mount point for removable drives, returns them in a list"""
+    usb_drives = []
+    for partition in psutil.disk_partitions():
+      # Exclude rootfs, the backup drive that happens to mount to /media/pi
+      if '/media' in partition.mountpoint and 'rootfs' not in partition.mountpoint:
+        usb_drives.append(str(partition.mountpoint))
+    return usb_drives
+
   def _update_sys_info(self, throttled=True) -> None:
     """Update current system information"""
     if self.status.info is None:
       raise Exception("No info generated, system in a bad state")
     self.status.info.online = self._online_cache.get(throttled)
+    self.status.info.connected_drives = self.get_usb_drives()
```

Comment:
Can you use a TimeBasedCache for this? This function gets called for every GET...

Reply:
See these for more info:

```
 self._online_cache = utils.TimeBasedCache(self._check_is_online, 5, 'online')
    self._latest_release_cache = utils.TimeBasedCache(self._check_latest_release, 3600, 'latest release')
```